packages/nvidia_nat_memmachine/tests/test_memory.py

```
@@ -0,0 +1,193 @@
+        async with memmachine_memory_client(config, mock_builder) as editor:
+            assert editor is not None
+            # Project creation should have been attempted
+            mock_memmachine_client.get_or_create_project.assert_called_once()
+
+
+
+
+async def test_memmachine_memory_client_config_validation():
+    """Test that MemMachineMemoryClientConfig validates required fields."""
+    # base_url is required
+    with pytest.raises(Exception):  # Pydantic validation error
```

Ruff (0.14.13)

Do not assert blind exception: `Exception`

(B017)